clang/test/CodeGen/cheri/cheri-asm-ptrtoint.c

```diff
@@ -0,0 +1,31 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 2
+// RUN: %riscv32_cheri_purecap_cc1 -S -emit-llvm -O1 -o - %s -verify | FileCheck %s --check-prefix=RV32
```

Comment:
-verify shouldn't really be used in CodeGen, that's for Sema. But what are you actually trying to test here? There wasn't a diagnostic emitted before?